src/main/java/luke/gui/MainWindow.java

```diff
@@ -0,0 +1,4 @@
+package luke.gui;
+
+public class MainWindow {
```

Comment:
Maybe you committed this accidentally, but if this is intentional you may want to add a comment as to why this class is blank.

Reply:
Oh, I was actually in the middle of setting up the gui. I will definitely make sure to double check  before I commit next time!